src/accel/EventAction.cc

```diff
@@ -17,7 +17,12 @@ namespace celeritas
 /*!
  * Construct with Celeritas shared data.
  */
-EventAction::EventAction() {}
+EventAction::EventAction(SPParams params, SPTransporter transport)
+    : params_(params), transport_(transport)
```

Comment:
It looks like nothing else in this classes uses the `Params`; do you anticipate it'll be needed?

Reply:
Nope! Removed.